tag-historian/nitaghistorian.yml

```diff
@@ -226,6 +232,9 @@ definitions:
         description: The path of the tag corresponding to the historical value.
         type: string
         example: system1.tag1
+      workspace:
+        description: The workspace Id of the tag
+        example: 0c80cf49-54e9-4e92-b117-3bfa574caa84
```

Comment:
Workspace also needs to be added to the output of the HTTP query routes (currently as part of HttpHistoricalValue, but there's a bug to move that up to the same level as type), right? Not sure if we would add it directly to the responses or to the TypeAndValues definition below.

Reply:
I have added the workspace to the response type